src/Playwright/Core/BrowserContext.cs

```diff
@@ -124,11 +126,9 @@ internal BrowserContext(ChannelOwner parent, string guid, BrowserContextInitiali
 
     public event EventHandler<IWorker> ServiceWorker;
 
-    public ITracing Tracing
-    {
-        get => _tracing;
-        set => throw new NotSupportedException();
-    }
+    public ITracing Tracing { get => _tracing; }
+
+    public IClock Clock { get => _clock; }
```

Comment:
Nit: Could be even shorter like the other ones:

```suggestion
    public ITracing Tracing => _tracing;

    public IClock Clock => _clock;
```